Source/MLX/DType.swift

```diff
@@ -90,6 +90,21 @@ public enum DType: Hashable {
         default: false
         }
     }
+
+    public var isInteger: Bool {
```

Comment:
These are used mostly by the bitwise operators where are not in this change, see https://github.com/ml-explore/mlx-c/issues/23